server.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, request, make_response, json
+#from find_melanoma import Melanoma
+import base64
+
+app = Flask(__name__)
+
+
+@app.route("/patient_classification", methods=['POST'])
+def patient_prediction():
+    data = request.json
+    prediction_result = {}
+    for k in data.keys():
+        encodeimage = data[k]
+        image_string = base64.b64decode(encodeimage)
+        image_result = open('{}_decode_image.jpg'.format(k), 'wb')
```

Comment:
Is `image_result` ever closed?  That would be good practice.  Could avoid having to explicitly closing this with a `with open() as ` construct.